bodhi/server/templates/update.html

```diff
@@ -161,7 +161,13 @@
         // note: these are actually single-item arrays, though this doesn't
         // seem to break anything
         var arch = result.data.arch;
+        if (arch === undefined){
+            arch = 'noarch';
```

Comment:
Interesting thing is that some of our tasks like `dist.rpmlint` don't fill out the `arch` field, and nothing breaks. Is this really necessary?

Reply:
Without this change and the one under I was getting a object whose first attribute was ``undefined`` making it rather hard to access what's underneath :)

Reply:
Ok apparently this is no longer needed, dropping it.